cmd/eks.go

```diff
@@ -77,6 +77,17 @@ var (
 		Name:  "issuer-url",
 		Usage: "(Required) URL of the OIDC Issuer for which we want to retrieve CA certificate thumbprints for.",
 	}
+
+	// Flags for cleaning up security group
+	securityGroupIDFlag = cli.StringFlag{
+		Name:  "security-group-id",
+		Usage: "ID of the Security Group created by EKS to manage EKS nodes.",
+	}
```

Comment:
NIT: I wonder if this should be a string slice? After seeing this, I am now thinking the ergonomics of the CLI is:

- Pass in SG IDs that you know you want to delete with `--security-group-id`, but it is not strictly required (this is a convenience feature). We probably want to support passing in multiple in case AWS decides to add more SGs to the mix.
- Regardless of passing in SG IDs, this command will always look up related SGs using tags, and delete those.

Reply:
There isn't a `StringSliceFlagRequiredE` in the gruntwork-cli/entrypoint/assertions.go. so I will use `urfave`, as in `	securityGroupIDs := cliContext.StringSlice(securityGroupIDFlag.Name)`

Looking at their [docs](https://github.com/urfave/cli/blob/master/docs/v2/manual.md#full-api-example), it doesn't look like they expect each one to be passed in like `--security-group-id security-123456 --security-group-id security-abcdef` etc, but rather like `--security-group-ids ["security-1233456", "security-abcdef"]`. But there isn't an explicit example. Also rather hard to locate one on the web. I'll just test this locally.

Reply:
Note that we are still on v1 of urfave/cli, so the relevant docs is https://github.com/urfave/cli/blob/master/docs/v1/manual.md#full-api-example, but `StringSlice` does expect each one to be passed in like that ([source](https://github.com/urfave/cli/issues/910)).

Reply:
I'm filing this change as a nice-to-have to add later.